Datadog/Datadog.xcodeproj/project.pbxproj

```diff
@@ -2979,12 +3010,17 @@
 		};
 		61133B97242393DE00786299 /* Debug */ = {
 			isa = XCBuildConfiguration;
+			baseConfigurationReference = 61569894256D0E9A00C6AADA /* Base.xcconfig */;
 			buildSettings = {
 				CODE_SIGN_STYLE = Automatic;
 				DEFINES_MODULE = YES;
 				DYLIB_COMPATIBILITY_VERSION = 1;
 				DYLIB_CURRENT_VERSION = 1;
 				DYLIB_INSTALL_NAME_BASE = "@rpath";
+				FRAMEWORK_SEARCH_PATHS = (
+					"$(inherited)",
+					"$(SRCROOT)/../Carthage/Build/iOS",
+				);
```

Comment:
i don't know if that works but what if we add `Khronos` as a Swift Package via Xcode's interface?
i guess that we may not need to change `framework search paths` and `make dependencies` step in `Makefile` (also we will be less dependent on `carthage` 🙌 )

wdyt?

Reply:
IDK if Carthage supports fetching `.xcodeproj` SPM dependencies before build (probably yes, as it's `xcodebuild's` job), but even if it does, this will be problematic / incorrect if the app is using another `Carthage` dependency which depends on `Kronos`. I can see two problems:
* version resolution: `Carthage` doesn't talk to `SPM` so they won't be able to resolve common `Kronos` version,
* conflicting symbols: our SPM's `Kronos` symbols may conflict with `Kronos` installed by `Carthage`.

Reply:
hmm, yes that's true. the host project will also crash at runtime as it won't have `Khronos` embedded in itself.
okay, so we will need to stick with `carthage` then: resolved